bundle/manifests/argocd-operator.clusterserviceversion.yaml

```diff
@@ -1733,6 +1733,7 @@ spec:
               containers:
               - args:
                 - --leader-elect
+                - --label-selector
```

Comment:
This command line arg can be removed as the Environment variable is set. Also what would be the behaviour if both command line argument `--label-selector` and the environment variable `ARGOCD_LABEL_SELECTOR` is used but with different values ? Which one would take more precedence ?

Reply:
@anandf @raghavi101 was there a resolution to this comment?

Reply:
@anandf correct me if I am wrong, for applying the label-selector the operator needs to be patched that can be done via setting the env var which means the flag option might become null and void. Referring the gitops-operator's [kuttl test](https://github.com/redhat-developer/gitops-operator/pull/607/files#diff-3ec4329326c83654616bbf537b95eb50b8aa36e4ae684bd6f1dff3cdfc0f1bc9R11)